Makefile

```diff
@@ -1,6 +1,13 @@
-.PHONY: style
+.PHONY: style lint
```

Comment:
Can you put these before each make target instead of here? That way it's easier to see which target has a `.PHONY` on it.

On a related note, `autoformat` should have `.PHONY: autoformat` :)

Reply:
Fixed.